toolset/setup/linux/languages/mono.sh

```diff
@@ -36,17 +35,20 @@ source $IROOT/mono.installing
 # start fresh
 rm -rf $MONO_HOME && mkdir -p $MONO_HOME
 
-# Download and extract debs
+# Download and extract debs.
 fw_apt_to_iroot mono-snapshot-$SNAPSHOT
 fw_apt_to_iroot mono-snapshot-$SNAPSHOT-assemblies mono-snapshot-$SNAPSHOT
 
+# Require sudo for some commands due to permission problem.
+# Other than that, this script will be stuck and will fail test.
+#
 # Simplify paths
-mv $MONO_HOME/opt/mono-*/* $MONO_HOME
+sudo mv $MONO_HOME/opt/mono-*/* $MONO_HOME
```

Comment:
Should not require sudo as the move is from `$IROOT/mono-snapshot-$SNAPSHOT/opt/mono-*/*` to `$IROOT/mono-snapshot-$SNAPSHOT`. Neither of these places require any special privileges for the testrunner user.